File: likes-service/src/main/java/com/ironhack/likesservice/model/Likes.java
package com.ironhack.likesservice.model;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Likes {
    @Id
    private Long id;
    private Long likes;
    private Long dislikes;

    public Likes() {
    }

    public Likes(Long id, Long likes, Long dislikes) {
        this.id = id;
        this.likes = likes;
        this.dislikes = dislikes;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getLikes() {
        return likes;
    }

    public void setLikes(Long likes) {
        this.likes = likes;
    }

    public Long getDislikes() {
        return dislikes;
    }

    public void setDislikes(Long dislikes) {
        this.dislikes = dislikes;
    }
}

File: recipes-app/recipes-angularapp/src/app/models/like/likes.ts
export class Likes {

    constructor(
        private _id: number,
        private _likes: number,
        private _dislikes: number
    ) {}

    public get dislikes(): number {
        return this._dislikes;
    }
    public set dislikes(value: number) {
        this._dislikes = value;
    }
    public get likes(): number {
        return this._likes;
    }
    public set likes(value: number) {
        this._likes = value;
    }
    public get id(): number {
        return this._id;
    }
    public set id(value: number) {
        this._id = value;
    }
}
